config/ldap.php

```diff
@@ -59,6 +59,7 @@
     'logging' => [
         'enabled' => env('LDAP_LOGGING', true),
         'channel' => env('LOG_CHANNEL', 'stack'),
+        'level' => env('LDAP_LOGGING_LEVEL', 'info'),
```

Comment:
Let's default to the application's default log level to stay consistent with the above `LOG_CHANNEL` env variable as well 👍 

```suggestion
        'level' => env('LOG_LEVEL', 'info'),
```